src/families/stellar/js-synchronization.js

```diff
@@ -0,0 +1,37 @@
+//@flow
+import { makeScanAccounts, makeSync, mergeOps } from "../../bridge/jsHelpers";
+import type { Account } from "../../types";
+import type { GetAccountShape } from "../../bridge/jsHelpers";
+import { fetchAccount, fetchOperations } from "./api";
+
+const getAccountShape: GetAccountShape = async (info) => {
+  const { id, address, initialAccount } = info;
```

Comment:
Did we confirm that migrating from libcore account will correctly works?

there is an implicit migration if users come with a libcore: account.
and i expect the output to be a `js:` id account. I fear a bit this do not happens here.
but maybe i fixed it generically in "makeSync" when i worked on the ethereum one